test/test.js

```diff
@@ -3301,6 +3303,24 @@ describe('Page', function() {
       }
     });
   });
+
+  describe('Response', function() {
+    it('specifies when response comes from cache and service worker', async({page, server}) => {
```

Comment:
Looks like `server` in `{page, server}` isn't being used :)

Reply:
done